api/v1beta1/kafkacluster_types.go

```diff
@@ -321,6 +321,8 @@ type EnvoyConfig struct {
 	// If not specified, the Envoy pods' priority is default to zero
 	// +optional
 	PriorityClassName string `json:"priorityClassName,omitempty"`
+	// EnableHealthCheckHttp10 is a toggle for adding HTTP1.0 support to Envoy health-check
```

Comment:
I would add `+optional` just to make it more explicit to the readers that this newly added field is optional
```suggestion
	// EnableHealthCheckHttp10 is a toggle for adding HTTP1.0 support to Envoy health-check
	// +optional
```